components/footer.jsx

```diff
@@ -2,6 +2,42 @@ var React = require('react');
 var Router = require('react-router');
```

Comment:
since you are on this file already, can we change these to `import` to make things look more unified?